Sources/Interaction/Style/InteractorStyleManipulator/index.js

```diff
@@ -239,11 +239,11 @@ function vtkInteractorStyleManipulator(publicAPI, model) {
     if (model.currentRenderer && model.currentManipulator) {
       // When an interaction is active, we should not change the renderer being
       // interacted with.
-    } else {
+    } else if (pos) {
       publicAPI.findPokedRenderer(pos.x, pos.y);
     }
 
-    if (model.currentManipulator) {
+    if (model.currentManipulator && model.currentRenderer && pos) {
```

Comment:
I'm guarding that here also in case we are missing the event and the currentRenderer.